README.md

```diff
@@ -99,6 +99,24 @@ If the toolbar `<script>` is accidentally included on your site, and `SentryTool
 
 #### Conditionally inserting script tag
 
-It's possible to dynamically insert the script tag inside an SPA app, prior to calling `SentryToolbar.init()`, so that only users who are elegible . See docs/conditional-script.md for example code. This will help reduce network traffic for your users because they do not have the credentials needed to login
+It's possible to dynamically insert the script tag inside an SPA app, prior to calling `SentryToolbar.init()`, so that only users who are eligible . See docs/conditional-script.md for example code. This will help reduce network traffic for your users because they do not have the credentials needed to login
 
 This example code will be eventually implemented as an NPM package, but for now it's something to be done manually.
+
+### Injecting the toolbar into a site
+
+To inject the toolbar into 'random' sites, configure and paste the following snippets into the console:
```

Comment:
Let's rephrase this to be like: temporarily into your own site

That kind of idea instead of saying 'random' literally.